examples/time_frequency/plot_single_trial_spectra.py

```diff
@@ -35,48 +35,21 @@
 include = []
 raw.info['bads'] += ['MEG 2443']  # bads
 
-# picks MEG gradiometers
-picks = mne.pick_types(raw.info, meg='grad', eeg=False, eog=True,
-                       stim=False, include=include, exclude='bads')
-
-epochs = mne.Epochs(raw, events, event_id, tmin, tmax, picks=picks, proj=True,
-                    baseline=(None, 0), reject=dict(grad=4000e-13, eog=150e-6))
-
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax, proj=True,
+                    baseline=(None, 0),
+                    reject=dict(grad=4000e-13, eog=150e-6))
 
 n_fft = 256  # the FFT size. Ideally a power of 2
-psds, freqs = compute_epochs_psd(epochs, fmin=2, fmax=200, n_fft=n_fft,
-                                 n_jobs=2)
-
-# average psds and save psds from first trial separately
-average_psds = psds.mean(0)
-average_psds = 10 * np.log10(average_psds)  # transform into dB
-some_psds = 10 * np.log10(psds[12])
-
-
-fig, (ax1, ax2) = plt.subplots(1, 2, sharex=True, sharey=True, figsize=(10, 5))
 
-fig.suptitle('Single trial power', fontsize=12)
+# Let's first check out all channel types by averaging across epochs.
+epochs.plot_psds(plot_kind=1, fmin=2, fmax=200, n_fft=n_fft, n_jobs=2)
```

Comment:
... and then this:

```Python
And this:

```Python
def plot_spectral_power_maps(info, psds, freqs, bands=None, show=True,
                             agg_fun=np.sum, unit=None, cmap='jet'):
    """Plot topomaps of the power spectral densities.
    Creates 5 plots grouped by bands:
    0 - 4 Hz   (Delta)
    4 - 8 Hz   (Theta)
    8 - 12 Hz  (Alpha)
    12 - 30 Hz (Beta)
    30 - 45 Hz (Gamma)


    Parameters
    ----------
    info : instance of mne.meas_info.Info
        The measurement info.
    psds : np.ndarray of float, shape (n_channels, n_freqs)
        Power spectral densities
    freqs : np.ndarray of float, shape (n_freqs)
        Frequencies used to compute psds.
    bands : list of tuple | None
        The lower and upper frequency and the name for that band. If None,
        (default) expands to:

        bands = [(0, 4, 'Delta'), (4, 8, 'Theta'), (8, 12, 'Alpha'),
                 (12, 30, 'Beta'), (30, 45, 'Gamma')]

    show : bool
        Whether to show the figure or not. Defaults to True.
    agg_fun : callable
        The function used to aggregate over frequencies. Defaults to np.sum.

    Returns
    -------
    fig : instance of matplotlib.pyplot.Figure
        The figure object.
    """
    import matplotlib.pyplot as plt

    pos = mne.channels.make_eeg_layout(info, exclude=[]).pos
    if bands is None:
        bands = [(0, 4, 'Delta'), (4, 8, 'Theta'), (8, 12, 'Alpha'),
                 (12, 30, 'Beta'), (30, 45, 'Gamma')]

    n_axes = len(bands)
    fig, axes = plt.subplots(1, n_axes, figsize=(1.5 * n_axes, 2))
    if n_axes == 1:
        axes = [axes]

    for (fmin, fmax, title), ax in zip(bands, axes):
        freq_mask = (fmin < freqs) & (freqs < fmax)
        data = agg_fun(psds[:, freq_mask], axis=1)
        plot_topomap_multi_cbar(data, pos, ax, title=title,
                                unit=unit, cmap=cmap)

    mne.viz.tight_layout(fig=fig)
    fig.canvas.draw()
    if show is True:
        plt.show()
    return fig

```